.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,63 @@
+name: Build Wheels
+
+on: [push, workflow_dispatch]
```

Comment:
Might just be because it's still draft but this probably doesn't need to run on every push. I'd suggest to run on every tag creation.